src/commands/info.ts

```diff
@@ -0,0 +1,60 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { CommandInteraction, EmbedBuilder, EmbedAuthorOptions,ColorResolvable, APIEmbedField } from 'discord.js';
+import * as config from "../config.json";
+import { Command } from '../types/command';
+import { PermissionLevel } from '../utils/permissions';
+
+
+const BotInfo: Command = {
+	permissionLevel: PermissionLevel.TEAM_MEMBER,
+
+	data: new SlashCommandBuilder()
+		.setName('info')
+		.setDescription('Responds with information about the bot.'),
+
+	async execute(interaction: CommandInteraction) {
+        
+        const username = interaction.guild?.members.cache.get(interaction.client.user?.id || "")?.nickname || interaction.client.user?.username || "";
```

Comment:
Surely you can just grab the username straight from the client

Would also be better to use `<@ID>` syntax and ignore special logic for nicknames